pages/resources/index.js

```diff
@@ -156,12 +155,75 @@ export const getStaticProps = async ({ params }) => {
     return a.kicker.localeCompare(b.kicker);
   });
 
+  const wrapper = await getProps(collections, recents);
   const navigation = await getNavigation();
 
   return {
     props: {
-      collections,
       navigation,
+      wrapper,
     },
   };
 };
+
+async function getProps(collections, recents) {
```

Comment:
I think this function could be refactored a little bit for clarity. It's not very human-friendly to read. (Looks more like a python function than a JS one ;) )